The config.h handles the gettext logic, make sure we use it instead of 
assuming gettext support is enabled.

http://bugs.gentoo.org/show_bug.cgi?id=84641

--- acl-2.2.27/setfacl/do_set.c
+++ acl-2.2.27/setfacl/do_set.c
@@ -36,7 +36,7 @@
 #include "sequence.h"
 #include "parse.h"
 
-#include <libintl.h>
+#include "config.h"
 
 
 extern const char *progname;
@@ -389,7 +388,7 @@
 			goto fail;
 		if (error > 0) {
 			acl_text = acl_to_any_text(acl, NULL, ',', 0);
-			fprintf(stderr, gettext("%s: %s: Malformed access ACL "
+			fprintf(stderr, _("%s: %s: Malformed access ACL "
 				"`%s': %s at entry %d\n"), progname, path_p,
 				acl_text, acl_error(error), which_entry+1);
 			acl_free(acl_text);
@@ -416,7 +416,7 @@
 			goto fail;
 		if (error > 0) {
 			acl_text = acl_to_any_text(default_acl, NULL, ',', 0);
-			fprintf(stderr, gettext("%s: %s: Malformed default ACL "
+			fprintf(stderr, _("%s: %s: Malformed default ACL "
 			                  "`%s': %s at entry %d\n"),
 				progname, path_p, acl_text,
 				acl_error(error), which_entry+1);
@@ -478,7 +478,7 @@
 			}
 		} else {
 			if (acl_entries(default_acl) != 0) {
-				fprintf(stderr, gettext(
+				fprintf(stderr, _(
 						"%s: %s: Only directories "
 						"can have default ACLs\n"),
 					progname, path_p);
